### ISSUE TITLE
Missing MavenCentral files

### ISSUE THREAD

**Issue by arek-kwapis** [Abstrat-Technologies/rustedanvil#16]
The following are missing from MavenCentral

org/ow2/asm/asm-debug-all/4.1/asm-debug-all-4.1.jar
org/ow2/asm/asm-debug-all/4.1/asm-debug-all-4.1-sources.jar
org/scala-lang/scala-library/2.10.2/scala-library-2.10.2.jar
org/scala-lang/scala-library/2.10.2/scala-library-2.10.2-sources.jar
org/scala-lang/scala-compiler/2.10.2/scala-compiler-2.10.2.jar
org/scala-lang/scala-compiler/2.10.2/scala-compiler-2.10.2-sources.jar
net/sf/jopt-simple/jopt-simple/4.5/jopt-simple-4.5.jar
net/sf/jopt-simple/jopt-simple/4.5/jopt-simple-4.5-sources.jar
org/lwjgl/lwjgl/lwjgl/2.9.0/lwjgl-2.9.0.jar
org/lwjgl/lwjgl/lwjgl/2.9.0/lwjgl-2.9.0-sources.jar
org/lwjgl/lwjgl/lwjgl_util/2.9.0/lwjgl_util-2.9.0.jar
org/lwjgl/lwjgl/lwjgl_util/2.9.0/lwjgl_util-2.9.0-sources.jar
net/sourceforge/argo/argo/2.25/argo-2.25.jar
net/sourceforge/argo/argo/2.25/argo-2.25-sources.jar
org/bouncycastle/bcprov-jdk15on/1.47/bcprov-jdk15on-1.47.jar
org/bouncycastle/bcprov-jdk15on/1.47/bcprov-jdk15on-1.47-sources.jar
com/google/guava/guava/14.0/guava-14.0.jar
com/google/guava/guava/14.0/guava-14.0-sources.jar
org/apache/commons/commons-lang3/3.1/commons-lang3-3.1.jar
org/apache/commons/commons-lang3/3.1/commons-lang3-3.1-sources.jar
commons-io/commons-io/2.4/commons-io-2.4.jar
commons-io/commons-io/2.4/commons-io-2.4-sources.jar
net/java/jinput/jinput/2.0.5/jinput-2.0.5.jar
net/java/jinput/jinput/2.0.5/jinput-2.0.5-sources.jar
net/java/jutils/jutils/1.0.0/jutils-1.0.0.jar
net/java/jutils/jutils/1.0.0/jutils-1.0.0-sources.jar
com/google/code/gson/gson/2.2.2/gson-2.2.2.jar
com/google/code/gson/gson/2.2.2/gson-2.2.2-sources.jar

**Comment by Unloaded123456** [comment]
Looks like closed because in Readme, it is marked as completed

**Comment by arek-kwapis** [comment]
Yes, the files are now documented (via this issue) so it's marked as done in the readme, but we need this list for Milestone 2. 

Edit: need to keep it in milestone 1 to count towards the milestone goal in github, can't assign 2 milestones... dumb github, dumb
